protokube/cmd/protokube/main.go

```diff
@@ -20,22 +20,48 @@ import (
 	"flag"
 	"fmt"
 	"github.com/golang/glog"
+	"github.com/spf13/pflag"
+	"k8s.io/kops/dns-controller/pkg/dns"
 	"k8s.io/kops/protokube/pkg/protokube"
+	"k8s.io/kubernetes/federation/pkg/dnsprovider"
 	"net"
 	"os"
 	"strings"
 )
 
+var (
```

Comment:
You could just put them in the same block 
`var(
          flags = .....
          Buildin......
     )   
`